blueprints/services/dagjson-over-http/client.go

```diff
@@ -281,33 +281,34 @@ func (c *{{.Type}}) {{.AsyncMethodDecl}} {
 func {{.ProcessReturnAsync}}(ctx {{.Context}}, ch chan<- {{.MethodReturnAsync}}, r {{.IOReader}}) {
```

Comment:
The HTTP response body is passed here as the reader but is never closed? I think we should pass a ReadCloser here instead of just a Reader, and defer close it, otherwise there will be resource leaks.

Reply:
(This issue isn't really related to your change, we can fix it in a different PR if you prefer, but it definitely should be fixed.)

Reply:
I can open a PR fixing it.